cflib/crtp/serialdriver.py

```diff
@@ -99,25 +90,29 @@ def connect(self, uri, linkQualityCallback, linkErrorCallback):
 
         # Prepare the inter-thread communication queue
         self.in_queue = queue.Queue()
-        self.out_queue = queue.Queue(1)
 
-        self.ser = serial.Serial(device, 512000, timeout=1)
+        self.cpx = CPX(UARTTransport(device, 576000))
+
+        self._thread = _CPXReceiveThread(self.cpx, self.in_queue,
+                                         linkErrorCallback)
+        self._thread.start()
 
-        # Launch the comm thread
-        self._receive_thread = _SerialReceiveThread(
-            self.ser, self.in_queue, linkQualityCallback, linkErrorCallback)
-        self._receive_thread.start()
-        self._send_thread = _SerialSendThread(
-            self.ser, self.out_queue, linkQualityCallback, linkErrorCallback)
-        self._send_thread.start()
+        # Switch the link bridge to CPX in the Crazyflie
+        self.cpx.sendPacket(CPXPacket(destination=CPXTarget.STM32,
+                                      function=CPXFunction.SYSTEM,
+                                      data=[0x21, 0x01]))
+        # Force client connect to true
+        self.cpx.sendPacket(CPXPacket(destination=CPXTarget.STM32,
+                                      function=CPXFunction.SYSTEM,
+                                      data=[0x20, 0x01]))
+
+        # TODO! These should be reset again!
```

Comment:
Make an issue for this

Reply:
With the intended usage this is not really needed (i.e connecting with a Raspberry Pi onboard)